compute/kubernetes/api-cli/creating-managing-kubernetes-lifecycle-cliv2.mdx

```diff
@@ -0,0 +1,322 @@
+---
+title: Creating and managing a Kubernetes Kapsule with CLI (v2)
+description: This page explains how to create and manage a Kubernetes Kapsule with CLI (v2)
+dates:
+  validation: 2021-08-05
+  posted: 2021-08-05
+---
+
+Scaleway's Kubernetes (K8s) Kapsules provide you with a managed environment to create, configure and run a cluster of preconfigured machines for containerized applications. You will be able to create clusters without the complexity of managing the infrastructure.
+
+You can create and manage your Kapsule clusters from the console, via the API or via the Scaleway Command Line Interface.
+
+## Scaleway Command Line Interface Overview
+
+
+[The Scaleway Command Line Interface (CLI)](https://github.com/scaleway/scaleway-cli) allows you to pilot your Scaleway infrastructure directly from your terminal, providing a faster way to administer and monitor your resources. Scaleway CLI is easy to set up and use, and an essential tool for operating efficiently in your cloud environment. The CLI provides many functionalities, including the ability to create and manage K8s Kapsules. 
+
+
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
+  - You have generated your [API key](/console/my-project/how-to/generate-api-key/)
+  - You have downloaded and installed the [Scaleway CLI](https://github.com/scaleway/scaleway-cli#scaleway-cli-v2)
+  - You have configured the [Scaleway CLI](/compute/bare-metal-cloud/api-cli/bare-metal-with-cli/#-Configuring-the-CLI)
+
+</Message>
+
+
+## Configuring the CLI
```

Comment:
I don't think this should be in the tutorial because in the pre-requirements we suppose that the Scaleway CLI is configured.